snet_cli/mpe_client_command.py

```diff
@@ -92,22 +90,21 @@ def _transform_call_params(self, params):
             rez[k_final] = v
         return rez
 
-    def _import_protobuf_for_channel(self):
-        channel_dir = self.get_channel_dir()
-        return import_protobuf_from_dir(channel_dir, self.args.method, self.args.service)
+    def _import_protobuf_for_service(self):
+        spec_dir = self.get_service_spec_dir(self.args.org_id, self.args.service_id)
+        return import_protobuf_from_dir(spec_dir, self.args.method, self.args.service)
 
-    def _call_server_via_grpc_channel(self, grpc_channel, nonce, amount, params, service_metadata):
-        stub_class, request_class, response_class = self._import_protobuf_for_channel()
+    def _call_server_via_grpc_channel(self, grpc_channel, channel_id, nonce, amount, params, service_metadata):
+        stub_class, request_class, response_class = self._import_protobuf_for_service()
 
         request  = request_class(**params)
         stub     = stub_class(grpc_channel)
         call_fn  = getattr(stub, self.args.method)
 
         if service_metadata["encoding"] == "json":
-            switch_to_json_payload_econding(call_fn, response_class)
+            switch_to_json_payload_encoding(call_fn, response_class)
```

Comment:
Can we not remove this? We dont use JSON encoding any more right

Reply:
But I'm not removing it.. I fix typo in "encoding" name.